es51x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.java

```diff
@@ -64,7 +64,9 @@
 
 public class ReadonlyRestPlugin extends Plugin
     implements ScriptPlugin, ActionPlugin, IngestPlugin, NetworkPlugin {
-
+  static {
```

Comment:
static? What about moving it to ReadonlyRestPlugin companion object?

Reply:
oh, I see it is a java class. I think new modules already have scala's ReadonlyRestPlugin class. Please, refactor is a little to be consistent in all modules.

Reply:
I'm not sure what exactly you mean. Rewrite to scala? Or invoke method in constructor?

Reply:
You can copy scala's ReadonlyRestPlugin to other modules and adapt it (adaptations will be trivial). Remove java implementations.